common/test/data/uploads/generic_csv.csv

```diff
@@ -0,0 +1,7 @@
+column_1,column_2
```

Comment:
authors note: added this file because removing the cohort csv's had collateral damage on the OwnLearnerProfilePageTest.